firefox-ios/firefox-ios-tests/Tests/ClientTests/Mocks/MockDataCleaner.swift

```diff
@@ -0,0 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+@testable import Client
+
+class MockCookiesClearable: Clearable {
```

Comment:
Shouldn't we make this `CookiesClearable` so you can avoid the label string etc.?

Reply:
Makes sense